src/material-experimental/mdc-chips/chips.scss

```diff
@@ -33,6 +33,13 @@
       outline: dotted 2px;
     }
   }
+
+  // Explicitly set ripple opacity to 0 when not focus to avoid always applying mdc ripple opacity.
+  &:not(:focus) {
+    .mdc-chip__ripple {
+      opacity: 0;
+    }
```

Comment:
This increases the specificity a lot and makes it harder to change the opacity to something else, e.g. if we needed a different one on hover. Based on [these styles](https://github.com/angular/components/blob/master/src/material-experimental/mdc-chips/chips.scss#L62), the element should be transparent by default. Maybe this indicates that there's a different issue someplace else?

Reply:
Changed to including different mixins instead of increasing specificity